tests/files/test_plots.py

```diff
@@ -0,0 +1,42 @@
+import json
+import pathlib
+
+import pandas as pd
+import yaml
+
+import zntrack
+import zntrack.group
+
+CWD = pathlib.Path(__file__).parent.resolve()
+
+
+class Plotting(zntrack.Node):
+    a: pd.DataFrame = zntrack.plots(
+        y="loss",
+        x="epoch",
+        x_label="Epoch",
+        y_label="Loss",
+        template="plotly_dark",
+        title="Loss over Epochs",
+    )
+
+
+def test_plotting(proj_path):
+    with zntrack.Project() as proj:
+        Plotting()
```

Comment:
should this work despite not saving anything to plotting.a?

Reply:
The tests in `test/files` are for testing the generated files only. In other words, the `proj.build()` whilst the actual graph execution is tested in the integration tests.